keyboards/craftwalk/readme.md

```diff
@@ -0,0 +1,15 @@
+# craftWalk
+
+<img src="https://github.com/sotoba/craftwalk/blob/images/craftwalk_main.jpg" width="640">
```

Comment:
dead image link

Reply:
oops
target repository was private, modified setting to public